newrelic/resource_newrelic_alert_channel.go

```diff
@@ -141,6 +173,19 @@ func resourceNewRelicAlertChannelRead(d *schema.ResourceData, meta interface{})
 
 	d.Set("name", channel.Name)
 	d.Set("type", channel.Type)
+
+	// extract headers from Configuration before we try and set it in the resource
+	if headers, ok := channel.Configuration["headers"]; ok {
+		d.Set("headers", headers)
+		delete(channel.Configuration, "headers")
+	}
+
+	// extract payload from Configuration before we try and set it in the resource
+	if payload, ok := channel.Configuration["payload"]; ok {
+		d.Set("payload", payload)
+		delete(channel.Configuration, "payload")
```

Comment:
I'm a little hesitant to do this `delete` as its a breaking change for anyone already using this behavior.